README.md

```diff
@@ -26,37 +26,38 @@ An [OpenAPI specification](https://github.com/OAI/OpenAPI-Specification) describ
 
 A Looker SDK has several parts:
 
-- **Looker API** OpenAPI specification (e.g., found at
-  `https://<your-looker-endpoint>:19999/api/3.1/swagger.json`, although this is still the Swagger 2.x representation)
+- The **Looker API**, described by an OpenAPI specification (e.g., the Swagger 2.x representation found at
+  `https://<your-looker-domain>:19999/api/4.0/swagger.json`). The 4.0 API is our current & stable API. As of June 2022, [3.x is deprecated](https://developers.looker.com/api/advanced-usage/version-3x-deprecation).
 
 - The **Looker API Docs viewer**, provided in the Looker web app directly from our version-specific OpenAPI specification, available on each Looker server instance.
```

Comment:
The API Docs viewer is gone now, and has been replaced by API Explorer, which is available to install for each instance and also on the developer portal

Reply:
Updated to API Explorer